### PR TITLE
Allow .git directory outside the project folder. Fix for forward slash on windows

### PR DESCRIPTION
**Pull request by kdeggelman**

### CODE REVIEW

File: src/commands.ts [hunk 1]
```diff
@@ -72,7 +73,7 @@ async function open ( file = false, page? ) {
 
     const editorPath = activeTextEditor.document.uri.fsPath;
 
-    filePath = editorPath ? editorPath.substring ( repopath.length + 1 ) : undefined;
+    filePath = editorPath ? editorPath.substring ( repopath.length + 1 ).replace( /\\/g, '/' ) : undefined;
```
Comment (kdeggelman) [inline src/commands.ts]: Part of me was hoping for a more robust fix. Maybe something utilizing https://nodejs.org/api/url.html#url_url_pathtofileurl_path

Comment (fabiospampinato) [reply src/commands.ts]: If you find a cleaner solution feel free to use it, I've personally usually resorted to the regex solution too.

File: src/commands.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import * as pify from 'pify';
 import * as vscode from 'vscode';
 import Config from './config';
 import Utils from './utils';
+import { pathToFileURL } from 'url';
```
Comment (fabiospampinato) [inline src/commands.ts]: This import seems unused, can you remove it?

Comment (kdeggelman) [reply src/commands.ts]: Good catch, removed.

**Comment by fabiospampinato** [comment]
Thanks, available in v1.2.0 👍

**Pull request merged**